sources/packages/web/src/views/aest/student/FileUploads.vue

```diff
@@ -64,7 +64,7 @@
       </DataTable>
     </content-group>
     <formio-modal-dialog
-      max-width="730"
+      :max-width="730"
```

Comment:
do we need line 6      ` class="m-1"`

Reply:
for line 31, when the data table is empty. are we not showing the icon??

Reply:
> for line 31, when the data table is empty. are we not showing the icon??

@hellolynn-tbtb  Can you please confirm if we need the empty svg cat image for empty datatable.

Reply:
@hellolynn-tbtb is on vacation. All others are showing the image, please wrap the component with the 
```
<toggle-content :toggled="!something.length">
```